BlueprintUI/Sources/Layout/Hidden.swift

```diff
@@ -0,0 +1,45 @@
+import CoreGraphics
+
+/// `Hidden` conditionally hides its wrapped element.
+///
+/// - Note: When an element is hidden, any elements within the wrapped element will be hidden.
+public struct Hidden: Element {
+    public var isHidden: Bool
+    public var wrappedElement: Element
+
+    public init(_ isHidden: Bool = true, wrapping element: Element) {
+        self.isHidden = isHidden
+        wrappedElement = element
+    }
+
+    public var content: ElementContent {
+        ElementContent(child: wrappedElement, layout: Layout(isHidden: isHidden))
+    }
+
+    public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
+        nil
+    }
+
+    private struct Layout: SingleChildLayout {
+        var isHidden: Bool
+
+        func measure(in constraint: SizeConstraint, child: Measurable) -> CGSize {
+            child.measure(in: constraint)
+        }
+
+        func layout(size: CGSize, child: Measurable) -> LayoutAttributes {
+            var attributes = LayoutAttributes(size: size)
+            attributes.isHidden = isHidden
+            return attributes
+        }
+    }
+}
+
+extension Element {
+    /// Conditionally hide the wrapped element.
+    ///
+    /// - Note: When an element is hidden, any elements within the wrapped element will be hidden.
+    public func hidden(_ hidden: Bool = true) -> Hidden {
```

Comment:
Similar Q: re `isHidden`?

Reply:
I also prefer hidden here; and it matches SwiftUI

Reply:
Agree with trying to match SwiftUI here (at least with the default `.hidden()` usage), but don't feel strongly.

Reply:
Hmm ok; SG. This will diverge from UIKit, but I guess that's fine!

Reply:
Does UIKit have a functional chaining syntax to set hidden on a view like this?

This isn't used for if checks where "is" reads better (although the wrapping view's properly is `isHidden`). IMO chaining `hidden()` vs `isHidden()` reads better.